core/DataAccess/ArchiveSelector.php

```diff
@@ -96,25 +101,17 @@ public static function getArchiveIdAndVisits(ArchiveProcessor\Parameters $params
             $minDatetimeArchiveProcessedUTC = Date::factory($minDatetimeArchiveProcessedUTC);
         }
 
-        if (!empty($minDatetimeArchiveProcessedUTC) && !is_object($minDatetimeArchiveProcessedUTC)) {
```

Comment:
@diosmosis was this removed on purpose?

Reply:
it's duplicate code from two weird merges. if you look right above it, the same code is there.